modules/mono/editor/bindings_generator.cpp

```diff
@@ -4889,8 +4889,14 @@ void BindingsGenerator::handle_cmdline_args(const List<String> &p_cmdline_args)
 	}
 
 	if (glue_dir_path.length()) {
-		handle_cmdline_options(glue_dir_path);
-		// Exit once done
+		if (Engine::get_singleton()->is_editor_hint() ||
+				Engine::get_singleton()->is_project_manager_hint()) {
```

Comment:
Makes me wonder if we should add complementary cases, either a `is_editor_or_project_manager_hint` or a `is_game/running_hint` or something like it, though I'm not sure how often this particular check is used, or the complement

Reply:
This is something I'd like to refactor one day, `main.cpp` and the whole logic to infer what mode to run based on context is very hacky.

We should change it so there's a lot more _intent_ on what to run (can still be implicit for the user but the code flow needs to be explicit), and then we can store which run mode we're using in an enum instead of these custom bools.